astropy/io/misc/asdf/__init__.py

```diff
@@ -25,4 +25,103 @@
 Documentation on the ASDF Standard can be found `here
 <https://asdf-standard.readthedocs.io>`__. Documentation on the ASDF Python
 module can be found `here <https://asdf.readthedocs.io>`__.
+
+Using ASDF With Table I/O
```

Comment:
The text above ("Astropy currently has no ability to read or write ASDF files itself. In order  to process ASDF files it is necessary to make use of the standalone **asdf** package. ") should be modified now, right?

More generally, maybe the technical discussion about schemas and tags should be moved down below and just start with reading and writing tables?

Reply:
Good idea. I want to make sure this this information doesn't get buried though, since I'm not convinced that the `Table` interface will be the primary interaction for most users of `astropy` and `asdf`, at least in the near term.